test/service/test_manifest.py

```diff
@@ -241,6 +241,29 @@ def sort_key(row: JSON) -> bytes:
         expected.sort(key=sort_key)
         self.assertEqual(expected, manifest)
 
+    def _assert_pfb(self,
+                    expected_schema: JSON,
+                    expected_entities: list[JSON],
+                    actual: Response):
+        """
+        Assert that the body of the given response contains a valid PFB manifest
+        matching the expected schema and contents, disregarding the order of the
+        PFB entities.
```

Comment:
I may have been confused during my last review. *If* the verbatim manifests have non-deterministic ordering, there should be a comment that explains the reason for the non-determinism. I suspect that because the file-based manifest generators don't apply a pagination stage to the request, the hits are returned in an order that is not immediately apparent, but still deterministic. Ideally, there would be no sorting prior to the comparison, either of the actual value or the expected one. If we have to sort to work around a non-determinism, we should sort both the actual and the expected value prior to comparing them. Either way, the method should not have side-effects on its arguments and the type hints should reflect that.